src/main/java/com/gengo/client/JsonHttpApi.java

```diff
@@ -452,7 +452,7 @@ private String httpDelete(HttpURLConnection con) throws GengoException
     private String httpPut(HttpURLConnection con, String query) throws GengoException
     {
         con.setDoOutput(true);
-        con.setRequestProperty("Content-Type", "text/plain");
+        con.setRequestProperty("Content-Type", "application/x-www-form-urlencoded");
```

Comment:
Don't you need charset like https://github.com/gengo/gengo-java/blob/master/src/main/java/com/gengo/client/JsonHttpApi.java#L254